gettsim/visualization.py

```diff
@@ -571,3 +331,246 @@ def _kth_order_successors(dag, node, order):
     if 1 <= order:
         for successor in dag.successors(node):
             yield from _kth_order_successors(dag, successor, order=order - 1)
+
+
+def plot_dag(
```

Comment:
Move this function again to the top as it is the function most interesting for users looking at the code.